packages/preview/moremath/0.1.0/moremath.typ

```diff
@@ -0,0 +1,72 @@
+// === GENERIC ===
+
+// A handy thing to slightly increase parentheses size. E.g.:
+// `big((x - y) - (y - z))` will make the external parentheses a bit bigger
+#let big(x) = math.lr(x, size: 150%)
```

Comment:
it is possible to add more variants like $\bigl( \Bigl( \biggl( \Biggl($ `\bigl( \Bigl( \biggl( \Biggl(` ?

Reply:
Certainly! I'll take a look into it.